### ISSUE TITLE
If I have bowtie in my conda environment, Can I keep bowtiePath empty in miRador.ini?

### ISSUE THREAD

**Issue by Ryvesselt** [rkweku/miRador#9]
![图片](https://github.com/rkweku/miRador/assets/62634905/1eba57c3-ea4a-451d-a72c-b7701186e992)

The Question is the same as other softwares like blastn.

**Comment by rkweku** [comment]
Yes, if the tools are always in your PATH then there is no need to specify the path to miRador. I typically ran it that way myself actually. Let me know if you have problems with this. I'll leave the issue open for now.
